.github/workflows/aarch64_macos_bazel.yml

```diff
@@ -0,0 +1,35 @@
+name: AArch64 MacOS Bazel
```

Comment:
Can you add this line on top ?
```
# ref: https://github.com/actions/runner-images
```

e.g. https://github.com/google/or-tools/blob/main/.github/workflows/amd64_linux_cmake_cpp.yml#L1

Reply:
done